src/App.css

```diff
@@ -1,3 +1,13 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
 .App {
-  text-align: center;
+  padding: 6rem 2rem;
+  background-color: #f5f6fa;
+}
+
```

Comment:
- [ ] You should also add live between progress and current chapter

![image](https://user-images.githubusercontent.com/69011963/173165254-4124d81b-f1d1-429d-a031-3f541b0f0464.png)